dango/testing/tests/safe.rs

```diff
@@ -24,40 +24,41 @@ fn safe() {
 
     // Onboard 4 users. Three will be the members of the Safe. The other will
     // play the role of an attacker.
-    let [mut member1, member2, member3, attacker] = ["member1", "member2", "member3", "attacker"]
-        .into_iter()
-        .map(|username| {
-            let user = TestAccount::new_random(username).predict_address(
-                contracts.account_factory,
-                codes.account_spot.to_bytes().hash256(),
-                true,
-            );
-
-            suite.execute(
-                &mut accounts.relayer,
-                contracts.ibc_transfer,
-                &mock_ibc_transfer::ExecuteMsg::ReceiveTransfer {
-                    recipient: user.address(),
-                },
-                Coins::one("uusdc", 100_000_000).unwrap(),
-            );
-
-            suite.execute(
-                &mut Factory::new(contracts.account_factory),
-                contracts.account_factory,
-                &account_factory::ExecuteMsg::RegisterUser {
-                    username: user.username.clone(),
-                    key: user.key,
-                    key_hash: user.key_hash,
-                },
-                Coins::new(),
-            );
+    let [mut member1, mut member2, member3, attacker] =
+        ["member1", "member2", "member3", "attacker"]
+            .into_iter()
+            .map(|username| {
+                let user = TestAccount::new_random(username).predict_address(
+                    contracts.account_factory,
+                    codes.account_spot.to_bytes().hash256(),
+                    true,
+                );
+
+                suite.execute(
+                    &mut accounts.relayer,
+                    contracts.ibc_transfer,
+                    &mock_ibc_transfer::ExecuteMsg::ReceiveTransfer {
+                        recipient: user.address(),
+                    },
+                    Coins::one("uusdc", 100_000_000).unwrap(),
+                );
+
+                suite.execute(
+                    &mut Factory::new(contracts.account_factory),
+                    contracts.account_factory,
+                    &account_factory::ExecuteMsg::RegisterUser {
+                        username: user.username.clone(),
+                        key: user.key,
+                        key_hash: user.key_hash,
+                    },
+                    Coins::new(),
+                );
```

Comment:
Why do we have diffs in the Safe test? Did commits from unrelated PRs mix in

Reply:
I just added the test here to prove the bug I found. Should have opened a separate PR